app.py

```diff
@@ -1,120 +0,0 @@
-
-import os
-import praw
-
-from flask import Flask, redirect, request, Response
-
-
-app = Flask(__name__)
-Reddit = praw.Reddit(user_agent='slack-reddit-v1 by /u/arjunblj')
-
-SORT_TYPES = {
-    'hot': 'get_hot',
-    'new': 'get_new',
-    'top': 'get_top',
-    'rising': 'get_rising'
-}
-
-
-def _create_response(text):
-    """Returns a utf-8 encoded text response.
-    """
-    return Response(text, content_type='text/plain; charset=utf-8')
-
-
-def _parse_options(options):
-    """Returns a Response after validating the input.
-    """
-    if options[1] in SORT_TYPES.keys():
-        sort_type = options[1]
-        if len(options) > 2:
```

Comment:
What is options[1]? And why len(options) > 2?

Assign them to variables for documentation (e.g. `descriptive_variable_name = options[1]`) or comment them. I think the former is preferable to the latter, though.